api3dao/vitepress-docs
# add reference on how to verify the source of a dAPI (#600)

* add reference on how to verify the source of a dAPI

* update type to address

* update title

* Made change to this page, see commit msg
1. Added mentions of Market for the parameters
2. Capitalize Beacon at start of sentence
3. Add flexEndTag to the bottom of the page.

* fixed broken links

---------
